Makefile

```diff
@@ -68,17 +68,22 @@ compile-native: $(OUTPUT_DIR)/libp2p_nif.so
 
 
 # Run an interactive terminal with the main supervisor setup.
-start: compile-native
+start: compile-native compile_port
 	iex -S mix phx.server
 
 # Run an interactive terminal with the main supervisor setup.
-iex: compile-native
+iex: compile-native compile_port
 	iex -S mix
 
 # Install mix dependencies.
 deps:
 	cd $(LIBP2P_DIR)/go_src; \
 	go get && go install
+	cd libp2p_port; \
+	go get && go install
+	mix escript.install hex protobuf
+	go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
```

Comment:
`go get && go install` was failing for me back when I didn't have the protobuf-generated files. The order of operations should be:
1. `go install google.golang.org/protobuf/cmd/protoc-gen-go@latest`
2. generate protobuf files
3. `go get && go install`

Reply:
Done, thanks!